conbench/entities/commit.py

```diff
@@ -403,7 +403,7 @@ def _parse_commit(commit):
         return {
             "parent": commit["parents"][0]["sha"] if commit["parents"] else None,
             "date": dateutil.parser.isoparse(commit_author["date"]),
-            "message": commit["commit"]["message"].split("\n")[0],
+            "message": commit["commit"]["message"].split("\n")[0][:240],
```

Comment:
Did we truncate these elsewhere before the recent changes? I imagine we had to have, right?

Reply:
No! Isn't that crazy? We've just never ingested a commit with a message whose first line is over 250 characters. The backfill exposed one.

Reply:
Huh! I wonder if maybe the previous endpoint we were using automatically trimmed them and we didn't realize? 

I find it really surprising that we would only just now bump into it, but it is possible!

Reply:
I just queried all commits to apache/arrow, and here's the distribution of length of the first line of commit messages. Only 4 commits ever were over 250 characters!

<img width="615" alt="image" src="https://user-images.githubusercontent.com/16600275/202554439-94cac38c-5d54-4cd0-8bf4-6c558e6fc850.png">